.travis-build.sh

```diff
@@ -73,6 +74,81 @@ if [ ! -z "$FORMATTING" ]; then
 		;;
 	esac
 fi
+
+# If we are pushing changes to the master branch,
+# open PR to syl20bnr/develop.spacemacs.org with Spacemacs
+# documentation exported as HTML and formatted with spacefmt
```

Comment:
Actually one of the goals for `develop.spacemacs.org` is to be up to date with the develop branch of 
Spacemacs (the other role is website related development waiting for a new major release).